examples/advanced/nlp-ner/requirements.txt

```diff
@@ -0,0 +1,7 @@
+nvflare
+torch
+torchvision
+tensorboard
+transformers
+pandas
+seqeval
```

Comment:
do we need a third party license for seqeval?

Reply:
I think so

Reply:
added to license and self-checkout performed